README.adoc

```diff
@@ -256,6 +257,15 @@ The following slide deck examples have been exported using DeckTape:
 |https://astefanutti.github.io/decktape/examples/new-es5-overloards.pdf[new-es5-overloards.pdf] (0.1MB)
 |===
 
+=== WebSlides
```

Comment:
I tried to find a place where it makes sense and isn't too prominent. But that place in the _commands_ section is literally the first place I had it before I started overthinking it... so that's probably a good place if you think so, too. :)